docs/.eleventy/layouts/base.njk

```diff
@@ -40,7 +40,7 @@
 html, body, .markdown-body {
   font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Arial,Noto Sans,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol,Noto Color Emoji; 
 }
-.header-logo, .header-snowpack,
+.header-logo, .header-snowpack, .header-snowpack-blurb,
```

Comment:
style nits: this font is actually pretty tough to read at smaller sizes. I'd vote to just use our regular body font for it